modules/okhttp-client/src/main/java/com/spotify/apollo/http/client/HttpMetricModule.java

```diff
@@ -0,0 +1,62 @@
+/*-
+ * -\-\-
+ * Spotify Apollo okhttp Client Module
+ * --
+ * Copyright (C) 2013 - 2021 Spotify AB
```

Comment:
Copyright should be just 2021, right?

Reply:
Hmm, I guess. this was autogenerated when I ran `mvn license:update-file-header`